komga-webui/src/components/dialogs/EditSeriesDialog.vue

```diff
@@ -389,9 +431,11 @@ export default Vue.extend({
     },
     modal(val) {
       !val && this.dialogCancel()
+      val && this.getThumbnails()
     },
     series(val) {
       this.dialogReset(val)
+      this.getThumbnails()
```

Comment:
i think you should pass the `val` as parameter, and use the one in the parameter. Else sometimes you could try to get `this.series` inside `getThumbnails` while it has changed already, but you would get the old one instead of the new one.